src/axom/inlet/examples/CMakeLists.txt

```diff
@@ -11,6 +11,11 @@
 #------------------------------------------------------------------------------
 
 if (LUA_FOUND)
```

Comment:
Should this be guarded on `if(SOL_FOUND)`?  It uses `LuaReader`.

Reply:
Please build and test this branch against a `C++11`  configuration that has LUA, but not SOL

Reply:
SOL is builtin in so she cant... unless she deletes the header from our repo which seems drastic.

Reply:
Got it, so  `SOL_FOUND` and `LUA_FOUND` should always have the same value?

Reply:
Not necessarily... for example if you give Axom a LUA_DIR, LUA_FOUND will be true but if you are C++11, SOL_FOUND will be false.  I guarded all the places that were causing problems though.